imap_processing/swe/l1a_swe.py

```diff
@@ -0,0 +1,183 @@
+import bitstring
+import numpy as np
+import xarray as xr
+
+from imap_processing import packet_definition_directory
+from imap_processing.swe import decom_swe
+
+
+def uncompress(cem_count):
```

Comment:
I'm not sure how often this is being used below, but you might want to consider adding a cache to this since it seems to be a direct lookup table calculation. Or just pre-computing the entire lookup table ahead of time:

```suggestion
@functools.cache
def uncompress(cem_count):
```

or since you know there is a maximum of 256 values that seems reasonable to store up front if you're going to be hitting it a lot.

`decompression_table = [uncompress(i) for i in range(256)]`